infrastructure/main.tf

```diff
@@ -80,6 +80,7 @@ module "recipe-database" {
   subscription       = var.subscription
   key_vault_rg       = "genesis-rg"
   key_vault_name     = "dtssharedservices${var.env}kv"
+  business_unit      = "SDS"
```

Comment:
should this be business area as per https://github.com/hmcts/cnp-module-postgres/pull/70#discussion_r677388756

Reply:
yes